### PR TITLE
Add methods to client for unpause and scale requests

### PR DESCRIPTION
**Pull request by bacchus123**

### CODE REVIEW

**Comment by ssalinas** [comment]
Thanks for the PR @bacchus123 ! LGTM

File: SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java [hunk 1]
```diff
@@ -323,9 +327,25 @@ private void addCredentials(HttpRequest.Builder requestBuilder) {
     return Optional.absent();
   }
 
+  private <T> Optional<T> put(String uri, String type, Optional<?> body, Optional<Class<T>> clazz) {
```
Comment (tpetr) [inline SingularityClient/src/main/java/com/hubspot/singularity/client/SingularityClient.java]: it looks like this method is unused, and is causing findbugs to barf. could you remove this method to make the build pass?

**Comment by tpetr** [comment]
Pulled out of `hs_staging` due to failing build via https://github.com/HubSpot/Singularity/commit/ba6bf5f85e8c3fd4c0e689f7449c5bfe34cc4146

**Comment by bacchus123** [comment]
this builds locally with 
mvn clean install fine so I'm not sure what I need to fix??

**Comment by tpetr** [comment]
build with `mvn clean verify` to ensure that checkstyle, findbug, etc. all run

**Comment by bacchus123** [comment]
I ran `mvn clean verify` and all projects completed successfully

**Comment by ssalinas** [comment]
Thanks for fixing @bacchus123 , I put this back into staging

**Pull request merged**